ResourceStatusSystem/Agent/CacheFeederAgent.py

```diff
@@ -2,6 +2,16 @@
 
   This agent feeds the Cache tables with the outputs of the cache commands.
 
+The following options can be set for the CacheFeederAgent.
+
+::
+
+  CacheFeederAgent
+  {
+    # Use a dedicated proxy to submit jobs to the WMS
+    shifterProxy = DataManager
```

Comment:
I am wondering whether we want to put in each and every agent those options that are valid for all of them ? 
Also, this should go in the `dirac.cfg` and in the `ConfigTemplate` if not already there